### ISSUE TITLE
unexpected behavior with seqratio

### ISSUE THREAD

**Issue by nehasrikn** [maxbachmann/Levenshtein#41]
@maxbachmann Quick question for you! When I freshly install the module and run the seqratio example from the documentation, I'm getting some unexpected behavior! Running this snippet: 

```
>> import Levenshtein
>> print(Levenshtein.seqratio(['newspaper', 'litter bin', 'tinny', 'antelope'], ['caribou', 'sausage', 'gorn', 'woody']))
>> !python3 --version
```

gives me: 

```
7.215178571428572
Python 3.8.13
```

which isn't the expected value according to the documentation!

**Comment by maxbachmann** [comment]
Thanks for reporting. I fixed this in `v0.20.6`

**Comment by nehasrikn** [comment]
great, thank you!
